otcextensions/sdk/dms/v1/group.py

```diff
@@ -78,3 +77,23 @@ def create(self, session, group):
             self._body.clean()
 
         return self
+
+    def ack(self, session, queue_obj, ids, status='success'):
```

Comment:
this way we limit messages only to 1 status value.
Isn't there possibility to address pairs of messages and statuses?